extension.js

```diff
@@ -88,63 +136,34 @@ function _enable() {
+    SETTINGS = extension.getSettings('org.gnome.shell.extensions.dash-to-panel');
+    DESKTOPSETTINGS = extension.getSettings('org.gnome.desktop.interface');
+    EXTENSION_UUID = extension.uuid
+    EXTENSION_PATH = extension.path
 
-    Main.layoutManager.startInOverview = !Me.settings.get_boolean('hide-overview-on-startup');
+    Main.layoutManager.startInOverview = !SETTINGS.get_boolean('hide-overview-on-startup');
 
-    if (Me.settings.get_boolean('hide-overview-on-startup') && Main.layoutManager._startingUp) {
+    if (SETTINGS.get_boolean('hide-overview-on-startup') && Main.layoutManager._startingUp) {
         Main.sessionMode.hasOverview = false;
         Main.layoutManager.connect('startup-complete', () => {
             Main.sessionMode.hasOverview = this._realHasOverview
         });
     }
 
-    panelManager = new PanelManager();
+    panelManager = new PanelManager.PanelManager();
 
     panelManager.enable();
     
-    Utils.removeKeybinding('open-application-menu');
-    Utils.addKeybinding(
-        'open-application-menu',
-        new Gio.Settings({ schema_id: WindowManager.SHELL_KEYBINDINGS_SCHEMA }),
-        () => {
-            if(Me.settings.get_boolean('show-appmenu'))
-                Main.wm._toggleAppMenu();
-            else
-                panelManager.primaryPanel.taskbar.popupFocusedAppSecondaryMenu();
-        },
-        Shell.ActionMode.NORMAL | Shell.ActionMode.POPUP
-    );
-}
-
-function disable(reset) {
-    panelManager.disable();
-    Me.settings.run_dispose();
-    Me.desktopSettings.run_dispose();
-
-    delete Me.settings;
-    panelManager = null;
-    
-    Utils.removeKeybinding('open-application-menu');
-    Utils.addKeybinding(
-        'open-application-menu',
-        new Gio.Settings({ schema_id: WindowManager.SHELL_KEYBINDINGS_SCHEMA }),
-        Main.wm._toggleAppMenu.bind(Main.wm),
-        Shell.ActionMode.NORMAL | Shell.ActionMode.POPUP
-    );
-
-    if (!reset) {
-        extensionSystem.disconnect(extensionChangedHandler);
-        delete global.dashToPanel;
-
-        // Re-enable Ubuntu Dock if it was disabled by dash to panel
-        if (disabledUbuntuDock && Main.sessionMode.allowExtensions) {
-            (extensionSystem._callExtensionEnable || extensionSystem.enableExtension).call(extensionSystem, UBUNTU_DOCK_UUID);
-        }
-
-        AppIcons.resetRecentlyClickedApp();
-    }
-
-    Main.sessionMode.hasOverview = this._realHasOverview;
+    // Utils.removeKeybinding('open-application-menu');
+    // Utils.addKeybinding(
+    //     'open-application-menu',
+    //     new Gio.Settings({ schema_id: WindowManager.SHELL_KEYBINDINGS_SCHEMA }),
+    //     () => {
+    //         if(SETTINGS.get_boolean('show-appmenu'))
+    //             Main.wm._toggleAppMenu();
+    //         else
+    //             panelManager.primaryPanel.taskbar.popupFocusedAppSecondaryMenu();
+    //     },
+    //     Shell.ActionMode.NORMAL | Shell.ActionMode.POPUP
+    // );
```

Comment:
open-application-menu and _toggleAppMenu are no longer available, do we have to replace this somehow?